xonsh/environ.py

```diff
@@ -629,1152 +629,1038 @@ def default_lscolors(env):
+    XONSH_INTERACTIVE = Var.with_default(
+        True,
+        "``True`` if xonsh is running interactively, and ``False`` otherwise.",
+        is_configurable=False,
+    )
+    XONSH_LOGIN = Var.with_default(
+        False,
+        "``True`` if xonsh is running as a login shell, and ``False`` otherwise.",
+        is_configurable=False,
+    )
+    XONSH_PROC_FREQUENCY = Var.with_default(
+        1e-4,
+        "The process frequency is the time that "
+        "xonsh process threads sleep for while running command pipelines. "
+        "The value has units of seconds [s].",
+    )
+    XONSH_SHOW_TRACEBACK = Var.with_default(
+        False,
+        "Controls if a traceback is shown if exceptions occur in the shell. "
+        "Set to ``True`` to always show traceback or ``False`` to always hide. "
+        "If undefined then the traceback is hidden but a notice is shown on how "
+        "to enable the full traceback.",
+    )
+    XONSH_SOURCE = Var.with_default(
+        "",
+        "When running a xonsh script, this variable contains the absolute path "
+        "to the currently executing script's file.",
+        is_configurable=False,
+    )
+    XONSH_STORE_STDIN = Var.with_default(
+        False,
+        "Whether or not to store the stdin that is supplied to the "
+        "``!()`` and ``![]`` operators.",
+    )
+    XONSH_STORE_STDOUT = Var.with_default(
+        False,
+        "Whether or not to store the ``stdout`` and ``stderr`` streams in the "
+        "history files.",
+    )
+    XONSH_STYLE_OVERRIDES = Var(
+        is_str_str_dict,
+        to_str_str_dict,
+        dict_to_str,
+        {},
+        "A dictionary containing custom prompt_toolkit/pygments style definitions.\n"
+        "The following style definitions are supported:\n\n"
+        "    - ``pygments.token.Token`` - ``$XONSH_STYLE_OVERRIDES[Token.Keyword] = '#ff0000'``\n"
+        "    - pygments token name (string) - ``$XONSH_STYLE_OVERRIDES['Token.Keyword'] = '#ff0000'``\n"
+        "    - ptk style name (string) - ``$XONSH_STYLE_OVERRIDES['pygments.keyword'] = '#ff0000'``\n\n"
+        "(The rules above are all have the same effect.)",
+    )
+    XONSH_TRACE_SUBPROC = Var.with_default(
+        False,
+        "Set to ``True`` to show arguments list of every executed subprocess command.",
+    )
+    XONSH_TRACEBACK_LOGFILE = Var(
+        is_logfile_opt,
+        to_logfile_opt,
+        logfile_opt_to_str,
+        None,
+        "Specifies a file to store the traceback log to, regardless of whether "
+        "``XONSH_SHOW_TRACEBACK`` has been set. Its value must be a writable file "
+        "or None / the empty string if traceback logging is not desired. "
+        "Logging to a file is not enabled by default.",
+    )
+    STAR_PATH = Var.no_default("env_path", pattern=re.compile(r"\w*PATH$"))
+    STAR_DIRS = Var.no_default("env_path", pattern=re.compile(r"\w*DIRS$"))
+
+
+class ChangeDirSetting(Xettings):
+    """``cd`` Behaviour"""
```

Comment:
```suggestion
    """``cd`` Behavior"""
```

We use US spelling in the rest of the docs, so I'd like to keep that consistent